pkgs/development/libraries/itk/4.x.nix

```diff
@@ -1,11 +1,12 @@
-{ stdenv, fetchurl, cmake, libX11, libuuid, xz, vtk_7 }:
+{ stdenv, fetchzip, cmake, libX11, libuuid, xz, vtk_7, Cocoa }:
 
 stdenv.mkDerivation rec {
-  name = "itk-4.13.2";
+  pname = "itk";
+  version = "4.13.3";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/itk/InsightToolkit-4.13.2.tar.xz";
-    sha256 = "19cgfpd63gqrvc3m27m394gy2d7w79g5y6lvznb5qqr49lihbgns";
+  src = fetchzip {
```

Comment:
Why fetchzip?

Reply:
Sure, switched to `fetchFromGitHub` like default ITK.